models/submissions.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
       type : Boolean
     },
     submissionNumber : Number,
+    scoringSystem : String,
```

Comment:
@deepakr123 - Add isRubricDriven key as well and in observation submissions

Reply:
resolved